### DISCUSSION TITLE
Does docusaurus supports GitHub syntax for note/warning blockquote?

### DISCUSSION THREAD

**Discussion by KSXGitHub** [facebook/docusaurus#9293] [Q&A] [answered]
https://github.com/orgs/community/discussions/16925

**Comment by Josh-Cena** [comment, marked as answer]
Nope, and we've seen some opposition to adding it: https://github.com/facebook/docusaurus/issues/7471

**Reply by slorber** [reply]
We don't support it but you could implement this yourself in userland if needed:
- Create a custom `blockquote` React component that inspect React children and look for `[!NOTE]`
- Swizzle MDXComponents and add your `blockquote` component: https://docusaurus.io/docs/markdown-features/react#mdx-component-scope
